.travis.yml

```
@@ -63,25 +58,15 @@ before_install:
 
     git config --global advice.detachedHead false
 
-    # Export tests environment variables
-    source tests/_ci/environment
-    export $(cut -d= -f1 tests/_ci/environment)
-
-    # Export build environment variables
-    export PHP_MAJOR="$(php -r 'echo phpversion();' | cut -d '.' -f 1)"
-    export PHP_MINOR="$(php -r 'echo phpversion();' | cut -d '.' -f 2)"
-    export PHP_VERNUM="$(php-config --vernum)"
-    export PHP_PEAR_PHP_BIN=$(phpenv which php)
+  - tests/_ci/export-variables.sh
```

You're missed `source` keyword:

```suggestion
  - source tests/_ci/export-variables.sh
```

Previously we used export in current shell. Now you're use subshell thus you have to _source_ this file.